Code/ForceField/MMFF/Params.cpp

```diff
@@ -55,18 +52,15 @@ MMFFAromCollection::MMFFAromCollection(const boost::uint8_t *mmffArom) {
 const boost::uint8_t defaultMMFFArom[] = {37, 38, 39, 44, 58, 59, 63, 64, 65,
                                           66, 69, 76, 78, 79, 80, 81, 82};
 
-class MMFFDefCollection *MMFFDefCollection::ds_instance = nullptr;
+class std::unique_ptr<MMFFDefCollection> MMFFDefCollection::ds_instance = nullptr;
 
 extern const std::string defaultMMFFDef;
 
 MMFFDefCollection *MMFFDefCollection::getMMFFDef(const std::string &mmffDef) {
-  if (!ds_instance) {
-    ds_instance = new MMFFDefCollection(mmffDef);
-  } else if (!mmffDef.empty()) {
-    delete ds_instance;
-    ds_instance = new MMFFDefCollection(mmffDef);
+  if (!ds_instance || !mmffDef.empty()) {
+    ds_instance = std::unique_ptr<MMFFDefCollection>(new MMFFDefCollection(mmffDef));
```

Comment:
Can't you just do this?
```suggestion
    ds_instance.reset(new MMFFDefCollection(mmffDef));
```

Reply:
Sure, I'll do.